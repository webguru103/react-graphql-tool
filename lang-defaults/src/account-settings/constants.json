[
  {
    "id": "first-name-too-long",
    "defaultMessage": "First name is too long, must be less than or equal to 40 characters"
  },
  {
    "id": "last-name-too-long",
    "defaultMessage": "Last name is too long, must be less than or equal to 40 characters"
  },
  {
    "id": "name-too-long",
    "defaultMessage": "Name is too long, first and last name must be each less than 40 characters"
  }
]