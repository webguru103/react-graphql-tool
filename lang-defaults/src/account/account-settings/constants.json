[
  {
    "id": "password-not-changed",
    "defaultMessage": "The old and new passwords are the same."
  }
]