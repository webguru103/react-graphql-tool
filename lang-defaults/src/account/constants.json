[
  {
    "id": "sign-up",
    "defaultMessage": "Sign Up"
  },
  {
    "id": "sign-up-as",
    "defaultMessage": "Sign up as"
  },
  {
    "id": "invited-to-sign-up",
    "defaultMessage": "You have been invited to sign up as:"
  },
  {
    "id": "activating-account",
    "defaultMessage": "Activating your account"
  },
  {
    "id": "account-activated",
    "defaultMessage": "Account activated"
  },
  {
    "id": "could-not-activate",
    "defaultMessage": "Could not activate account"
  },
  {
    "id": "please-wait",
    "defaultMessage": "Please wait..."
  },
  {
    "id": "successfully-activated",
    "defaultMessage": "Your account has been successfully activated. You can now log in."
  },
  {
    "id": "email-already-registered",
    "defaultMessage": "You already have a DealTap account. Confirm your password to continue."
  },
  {
    "id": "email-already-taken",
    "defaultMessage": "This email is already taken. Please try another one."
  },
  {
    "id": "empty-email",
    "defaultMessage": "The email is empty"
  },
  {
    "id": "invalid-email",
    "defaultMessage": "The email is invalid"
  },
  {
    "id": "network-error",
    "defaultMessage": "Something went wrong, please try again later..."
  },
  {
    "id": "wrong-username-password",
    "defaultMessage": "Email or password is wrong"
  },
  {
    "id": "email-empty",
    "defaultMessage": "Please input your email address"
  },
  {
    "id": "email-doesnt-exist",
    "defaultMessage": "Email address does not exist"
  },
  {
    "id": "email-resent",
    "defaultMessage": "The email has been sent again. Please check your email"
  },
  {
    "id": "request-error",
    "defaultMessage": "Sorry, there was an error processing your request!"
  },
  {
    "id": "request-being-processed",
    "defaultMessage": "Your request is being processed."
  },
  {
    "id": "brokerage-accepted",
    "defaultMessage": "Invitation accepted"
  },
  {
    "id": "could-not-accept-brokerage",
    "defaultMessage": "Invitation not accepted"
  },
  {
    "id": "brokerage-rejected",
    "defaultMessage": "Invitation rejected"
  },
  {
    "id": "could-not-reject-brokerage",
    "defaultMessage": "Invitation not rejected"
  },
  {
    "id": "could-not-find-invite",
    "defaultMessage": "No invite found."
  },
  {
    "id": "link-expired",
    "defaultMessage": "It appears this link has already expired."
  },
  {
    "id": "joined-brokerage",
    "defaultMessage": "Joined Brokerage Office"
  },
  {
    "id": "failed-to-join-brokerage",
    "defaultMessage": "Failed To Join Brokerage Office"
  },
  {
    "id": "already-joined-brokerage",
    "defaultMessage": "Already Joined a Brokerage Office"
  },
  {
    "id": "already-joined-brokerage-message",
    "defaultMessage": "It appears you have already joined a brokerage office as an agent."
  },
  {
    "id": "already-granted-permission",
    "defaultMessage": "Already granted permission"
  },
  {
    "id": "already-have-access",
    "defaultMessage": "Already Have Access"
  },
  {
    "id": "already-have-access-message",
    "defaultMessage": "It appears you already have access to Control Panel. Please log in instead."
  },
  {
    "id": "permission-granted",
    "defaultMessage": "Permission Granted"
  },
  {
    "id": "failed-to-grant-permission",
    "defaultMessage": "Failed to Grant Permission"
  },
  {
    "id": "get-agent-panel-access",
    "defaultMessage": "Get Agent Panel Access"
  },
  {
    "id": "get-agent-panel-access-message",
    "defaultMessage": "We already have your information since you have a DealTap account. Click confirm to get agent panel access."
  },
  {
    "id": "agent-identity-added",
    "defaultMessage": "Agent Identity Added"
  },
  {
    "id": "agent-identity-added-message",
    "defaultMessage": "You can now access DealTap's agent panel."
  },
  {
    "id": "account-created",
    "defaultMessage": "Account Created"
  },
  {
    "id": "account-created-message",
    "defaultMessage": "Your account has been created."
  },
  {
    "id": "failed-to-reject",
    "defaultMessage": "Failed to reset password"
  },
  {
    "id": "password-reset-success-title",
    "defaultMessage": "Password has been reset"
  },
  {
    "id": "password-reset-success-message",
    "defaultMessage": "Please log in to continue."
  },
  {
    "id": "reset-password",
    "defaultMessage": "Reset Password"
  }
]