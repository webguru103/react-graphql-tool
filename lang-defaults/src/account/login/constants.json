[
  {
    "id": "account-not-verified",
    "defaultMessage": "We noticed you still haven't verified your email address. We have sent another verification email. Please check your inbox."
  },
  {
    "id": "user-not-exists",
    "defaultMessage": "Sorry, we couldnâ€™t recognize this email"
  },
  {
    "id": "email-not-valid",
    "defaultMessage": "Please input a valid email address."
  },
  {
    "id": "email-empty",
    "defaultMessage": "Please input your email address"
  },
  {
    "id": "verify-email",
    "defaultMessage": "Verify your email"
  },
  {
    "id": "network-message",
    "defaultMessage": "Some error occurred, try again later"
  }
]