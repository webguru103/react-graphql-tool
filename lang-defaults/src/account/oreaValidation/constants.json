[
  {
    "id": "cannot-reach-title",
    "defaultMessage": "We couldn't reach OREA"
  },
  {
    "id": "make-request-msg",
    "defaultMessage": "In order to continue,\n    we need to verify your identity with OREA.\n    Please click the button below to verify\n    your identity."
  },
  {
    "id": "first-login-msg-accent",
    "defaultMessage": "In order to use OREA forms,"
  },
  {
    "id": "first-login-msg",
    "defaultMessage": "you need to verify your identity with OREA.\n    Please click the button below to verify\n    your identity."
  },
  {
    "id": "make-request-title",
    "defaultMessage": "Identity verification"
  },
  {
    "id": "cannot-reach-msg",
    "defaultMessage": "Sorry, but we were not able to reach\n    OREA at this moment. You can go back and\n    try again, or verify later in your account settings."
  },
  {
    "id": "do-it-later",
    "defaultMessage": "Do It Later"
  },
  {
    "id": "try-again",
    "defaultMessage": "Try Again"
  },
  {
    "id": "do-it-later-msg",
    "defaultMessage": "You can validate with OREA anytime in the Account Settings page."
  },
  {
    "id": "ok-button",
    "defaultMessage": "OK"
  },
  {
    "id": "cancel-button",
    "defaultMessage": "Cancel"
  },
  {
    "id": "do-it-later-button",
    "defaultMessage": "I'll do it later."
  },
  {
    "id": "go-to-orea",
    "defaultMessage": "Go to OREA validation page"
  },
  {
    "id": "ver-failure-msg",
    "defaultMessage": "Sorry, but we were not able to verify your identity with OREA. Please check your information and try again."
  },
  {
    "id": "ver-failure-title",
    "defaultMessage": "Verification failed"
  },
  {
    "id": "ver-sucess-msg",
    "defaultMessage": "Your identity has been successfully verified through OREA."
  },
  {
    "id": "ver-sucess-title",
    "defaultMessage": "OREA verificiation successful"
  },
  {
    "id": "done-button",
    "defaultMessage": "Done"
  }
]