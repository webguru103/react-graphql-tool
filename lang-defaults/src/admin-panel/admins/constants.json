[
  {
    "id": "request-error",
    "defaultMessage": "Sorry, there was an error processing your request!"
  },
  {
    "id": "success-invite",
    "defaultMessage": "The admin has been successfully invited"
  }
]