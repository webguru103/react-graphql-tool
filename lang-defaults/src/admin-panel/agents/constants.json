[
  {
    "id": "successfuly-sent-invite",
    "defaultMessage": "Invite has been sent"
  },
  {
    "id": "successfuly-sent-invites",
    "defaultMessage": "Successfuly sent {amount} invites"
  },
  {
    "id": "no-emails",
    "defaultMessage": "Please input an email"
  },
  {
    "id": "invalid-emails",
    "defaultMessage": "Please input valid emails separated with comma or return"
  },
  {
    "id": "request-error",
    "defaultMessage": "Sorry, there was an error processing your request!"
  },
  {
    "id": "couldnot-send-invite",
    "defaultMessage": "Could not send invite to the following agents:"
  }
]