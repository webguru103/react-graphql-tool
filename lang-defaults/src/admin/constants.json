[
  {
    "id": "empty-email",
    "defaultMessage": "Please input an email"
  },
  {
    "id": "empty-first-name",
    "defaultMessage": "Please input a first name"
  },
  {
    "id": "empty-last-name",
    "defaultMessage": "Please input a last name"
  },
  {
    "id": "request-error",
    "defaultMessage": "Sorry, there was an error processing your request!"
  }
]