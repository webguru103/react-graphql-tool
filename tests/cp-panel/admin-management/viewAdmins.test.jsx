/* eslint import/no-extraneous-dependencies: 0 */

import * as React from 'react';
import 'jest-dom/extend-expect';
import { render, cleanup, waitForElement } from 'react-testing-library';
import { withContext } from '../../test-helpers/contextWrapper';
import { ViewAdminsC } from '../../../src/control-panel/admin-management/viewAdmins';
import { DEBOUNCE_TIMEOUT } from '../../../src/shared/table/constants';

const mockResponse = {
  data: {
    admins: [
      {
        id: 104,
        email: 'abc1@gmail.com',
        firstName: 'Jacey',
        lastName: 'GreenHolt',
        phone: '1234567891',
        createdAt: '2013-02-08',
        loggedInAt: '2013-02-08',
        brokerageAclsByUserId: {
          nodes: [
            {
              id: 500,
              brokerageName: 'Brokerage Name',
            },
          ],
        },
      },
    ],
  },
};

describe('View Admins', () => {
  afterEach(cleanup);
  const Component = withContext(ViewAdminsC);

  test('renders', async (done) => {
    const fetchUsers = jest.fn(() => new Promise(resolve => resolve(mockResponse)));
    const { container, getByText } = render(<Component
      loading={false}
      onRefetch={fetchUsers}
    />);

    await waitForElement(() => getByText('Admin'));
    setTimeout(() => {
      expect(container).toMatchSnapshot();
      done();
    }, DEBOUNCE_TIMEOUT + 50);
  });

  test('fetch users upon mounting', async (done) => {
    const fetchUsers = jest.fn(() => new Promise(resolve => resolve(mockResponse)));
    const { getByText } = render(<Component
      loading={false}
      onRefetch={fetchUsers}
    />);

    await waitForElement(() => getByText('Admin'));
    setTimeout(() => {
      expect(fetchUsers).toHaveBeenCalledTimes(1);
      done();
    }, DEBOUNCE_TIMEOUT + 50);

  });
});
