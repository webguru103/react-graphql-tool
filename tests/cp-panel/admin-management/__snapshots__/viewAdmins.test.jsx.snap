// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Admins renders 1`] = `
<div>
  <div
    class="ReactTable -highlight"
  >
    <div
      class="rt-table"
      role="grid"
    >
      <div
        class="rt-thead -header"
        style="min-width: 500px;"
      >
        <div
          class="rt-tr"
          role="row"
        >
          <div
            class="rt-th rt-resizable-header -cursor-pointer"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <div
              class="rt-resizable-header-content"
            >
              Admin
              <button
                aria-label="Filter"
                class="MuiButtonBase-root-7 MuiIconButton-root-1"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
            </div>
            <div
              class="rt-resizer"
            />
          </div>
          <div
            class="rt-th rt-resizable-header -cursor-pointer"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <div
              class="rt-resizable-header-content"
            >
              Phone
            </div>
            <div
              class="rt-resizer"
            />
          </div>
          <div
            class="rt-th rt-resizable-header -cursor-pointer"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <div
              class="rt-resizable-header-content"
            >
              Brokerage Office(s)
              <button
                aria-label="Filter"
                class="MuiButtonBase-root-7 MuiIconButton-root-1"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
            </div>
            <div
              class="rt-resizer"
            />
          </div>
          <div
            class="rt-th rt-resizable-header -cursor-pointer"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <div
              class="rt-resizable-header-content"
            >
              Created At
            </div>
            <div
              class="rt-resizer"
            />
          </div>
          <div
            class="rt-th rt-resizable-header -cursor-pointer"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <div
              class="rt-resizable-header-content"
            >
              Last Login
            </div>
            <div
              class="rt-resizer"
            />
          </div>
        </div>
      </div>
      <div
        class="rt-thead -filters"
        style="display: none; min-width: 500px;"
      >
        <div
          class="rt-tr"
          role="row"
        >
          <div
            class="rt-th"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <input />
          </div>
          <div
            class="rt-th"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          />
          <div
            class="rt-th"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          >
            <input />
          </div>
          <div
            class="rt-th"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          />
          <div
            class="rt-th"
            role="columnheader"
            style="flex: 100 0 auto; width: 100px;"
            tabindex="-1"
          />
        </div>
      </div>
      <div
        class="rt-tbody"
        style="min-width: 500px;"
      >
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -odd"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -even"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -odd"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -even"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -odd"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -even"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -odd"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -even"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -odd"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
        <div
          class="rt-tr-group"
          role="rowgroup"
        >
          <div
            class="rt-tr -padRow -even"
            role="row"
          >
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
            <div
              class="rt-td"
              role="gridcell"
              style="flex: 100 0 auto; width: 100px;"
            >
              <span>
                 
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="pagination-bottom"
    >
      <div
        class="Pagination-paginationContainer-19"
      >
        <div
          class="Pagination-pagination-20"
        >
          <div
            class="MuiToolbar-root-32 MuiToolbar-regular-34 MuiToolbar-gutters-33 MuiTablePagination-toolbar-23"
          >
            <div
              class="MuiTablePagination-spacer-24"
            />
            <span
              class="MuiTypography-root-36 MuiTypography-caption-46 MuiTablePagination-caption-25"
            >
              Rows per page:
            </span>
            <div
              class="MuiInputBase-root-69 MuiTablePagination-input-29"
            >
              <div
                class="MuiSelect-root-62 MuiTablePagination-selectRoot-26"
              >
                <div
                  aria-haspopup="true"
                  aria-pressed="false"
                  class="MuiSelect-select-63 MuiTablePagination-select-27 Pagination-paginationSelect-21 MuiSelect-selectMenu-66 MuiInputBase-input-79"
                  data-testid="select"
                  role="button"
                  tabindex="0"
                >
                  10
                </div>
                <input
                  type="hidden"
                  value="10"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-10 MuiSelect-icon-68 MuiTablePagination-selectIcon-28"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </div>
            </div>
            <span
              class="MuiTypography-root-36 MuiTypography-caption-46 MuiTablePagination-caption-25"
            >
              0-0 of 0
            </span>
            <div
              class="PaginationActions-root-90"
            >
              <button
                aria-label="First Page"
                class="MuiButtonBase-root-7 MuiButtonBase-disabled-8 MuiIconButton-root-1 MuiIconButton-disabled-5"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M18.41 16.59L13.82 12l4.59-4.59L17 6l-6 6 6 6zM6 6h2v12H6z"
                    />
                    <path
                      d="M24 24H0V0h24v24z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
              <button
                aria-label="Previous Page"
                class="MuiButtonBase-root-7 MuiButtonBase-disabled-8 MuiIconButton-root-1 MuiIconButton-disabled-5"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
                    />
                    <path
                      d="M0 0h24v24H0V0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
              <button
                aria-label="Next Page"
                class="MuiButtonBase-root-7 MuiButtonBase-disabled-8 MuiIconButton-root-1 MuiIconButton-disabled-5"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                    />
                    <path
                      d="M0 0h24v24H0V0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
              <button
                aria-label="Last Page"
                class="MuiButtonBase-root-7 MuiButtonBase-disabled-8 MuiIconButton-root-1 MuiIconButton-disabled-5"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiIconButton-label-6"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-10"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5.59 7.41L10.18 12l-4.59 4.59L7 18l6-6-6-6zM16 6h2v12h-2z"
                    />
                    <path
                      d="M0 0h24v24H0V0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="rt-noData"
    >
      <span>
        Nothing to display!
      </span>
    </div>
    <div
      class="-loading"
    >
      <div
        class="-loading-inner"
      >
        <div
          class="MuiCircularProgress-root-91 MuiCircularProgress-colorPrimary-94 MuiCircularProgress-indeterminate-93"
          role="progressbar"
          style="width: 40px; height: 40px;"
        >
          <svg
            class="MuiCircularProgress-svg-96"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle-97 MuiCircularProgress-circleIndeterminate-99"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
